### PR TITLE
Updated punctuations

### PR DESCRIPTION
**Pull request by kannansuresh**
Updated punctuation and some phrasing.

### CODE REVIEW

**Review by KTibow — APPROVED** [review]

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-modernist
```
Comment (sbmueller) [inline _config.yml:1]: Please don't change the theme - it is intentionally kept simple and neutral. Themes are always subjective so I'm not open to discuss theme changes.

File: index.md [hunk 2]
```diff
@@ -1,73 +1,52 @@
...
+# Please Don't Just Say Hello in Chat
 
 ## TL;DR
 
-Modern chat communication should be done in a way that saves time for all
-participants and enables an efficient, asynchronous exchange of information.
-**Don't expect an answer if you just write "Hello" in chat**. Instead, phrase a
-message that allows your chat partners to provide a meaningful response as soon
-as they allocate time to read and answer your message.
+Modern chat communication should be done in a way that saves time for all participants and enables an efficient, asynchronous exchange of information. Don’t expect an answer if you just write “Hello” in chat. Instead, phrase a message that allows your chat partners to provide a meaningful response as soon as they allocate time to read and answer your message.
```
Comment (sbmueller) [inline index.md:5]: Please keep the line wrap at 79 characters. This helps readability in plain text. Without wrap, horizontal *and* vertical scrolling as well as more eye movement is necessary.

File: index.md [hunk 3]
```diff
@@ -1,73 +1,52 @@
...
 2010-07-19 12:32:15 co-worker: Hello.
 # CO-WORKER WAITS WHILE YOU PHRASE YOUR QUESTION
 2010-07-19 12:34:01 you: I'm working on [something] and I'm trying to do [etc...]
 2010-07-19 12:35:21 co-worker: Oh, that's [answer...]
-</pre>
+```
 
-It's as if you called someone on the phone and said "Hi!" and then put them on
-hold!
+It’s as if you called someone on the phone and said “Hi!” and then put them on hold!
```
Comment (sbmueller) [inline index.md:17]: I'm not sure why to replace all `"` by `“`. The former is valid punctuation. I reckon it is more difficult to incorporate follow-up changes when sticking to this convention, so I'd rather reject it. Also applies to all the other changes in this PR.

**Review by sbmueller — CHANGES_REQUESTED** [review]
Hi, sorry for my late reply.
The only improvements I see in this PR are one wording change and one capitalization change. I'm fine merging those. The other changes, I am not convinced they are necessary.

**Comment by kannansuresh** [comment]
This was not intentional. This pull request was placed by mistake. Sorry for the inconvenience.
